examples/api/e2e/playwright.config.ts

```diff
@@ -46,7 +46,7 @@ export default defineConfig({
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: "pnpm preview:worker",
+    command: "pnpm preview:worker --inspector-port 9330",
```

Comment:
I think the correct place to fix this is in the `package.json` `"dev:worker"` scripts, where we already hard code `--port` for each of these examples.

Reply:
good point 🙂 

updated 👍